test/parallel/test-runner-cli.js

```diff
@@ -56,23 +56,19 @@ const testFixtures = fixtures.path('test-runner');
 
 {
   // User specified files that don't match the pattern are still run.
-  const args = ['--test', testFixtures, join(testFixtures, 'index.js')];
-  const child = spawnSync(process.execPath, args);
+  const args = ['--test', join(testFixtures, 'index.js')];
+  const child = spawnSync(process.execPath, args, { cwd: testFixtures });
 
   assert.strictEqual(child.status, 1);
   assert.strictEqual(child.signal, null);
   assert.strictEqual(child.stderr.toString(), '');
   const stdout = child.stdout.toString();
   assert.match(stdout, /not ok 1 - .+index\.js/);
-  assert.match(stdout, /ok 2 - this should pass/);
-  assert.match(stdout, /not ok 3 - this should fail/);
-  assert.match(stdout, /ok 4 - .+subdir.+subdir_test\.js/);
-  assert.match(stdout, /ok 5 - this should pass/);
```

Comment:
Removed this, as this is on main and was like that at the time of the original PR